packages/lab/src/slider/Slider.tsx

```diff
@@ -75,99 +47,55 @@ export const Slider = forwardRef<HTMLDivElement, SliderProps>(function Slider(
     window: targetWindow,
   });
 
-  const trackRef = useRef<HTMLDivElement>(null);
-
-  const [value, setValue] = useControlled<SliderValue>({
+  const [value, setValue] = useControlled<number>({
     controlled: valueProp,
     default: defaultValue,
     name: "Slider",
     state: "Value",
   });
 
-  const updateValueItem = useValueUpdater(pushable, pushDistance, min, max);
-
-  const trackStyle = useMemo(
-    () => createTrackStyle(min, max, value),
-    [min, max, value]
-  );
-
-  const valueLength = Array.isArray(value) ? value.length : 1;
-
-  const handleStyles: CSSProperties[] = useMemo(
-    () => createHandleStyles(valueLength),
-    [valueLength]
-  );
-
-  const onMouseDown = useSliderMouseDown(
-    trackRef,
-    value,
-    min,
-    max,
-    step,
-    updateValueItem,
-    setValue,
-    onChange
-  );
-
-  const onKeyDown = useSliderKeyDown(
-    value,
-    min,
-    max,
-    pageStep,
-    step,
-    updateValueItem,
-    setValue,
-    onChange
-  );
-
   return (
-    <div
-      className={clsx(
-        withBaseName(),
-        {
-          [withBaseName("disabled")]: disabled,
-        },
-        className
-      )}
-      ref={ref}
-      onKeyDown={disabled ? undefined : onKeyDown}
-      aria-label={`${label} slider from ${min} to ${max}`}
-      role="group"
+    <SliderContext.Provider
+      value={{
+        value,
+        min,
+        max,
+        step,
+        setValue,
+        onChange,
+        ariaLabel,
+      }}
     >
-      {label !== undefined ? (
-        <div className={withBaseName("label")}>{label}</div>
-      ) : null}
       <div
-        className={withBaseName("clickable")}
-        onMouseDown={disabled ? undefined : onMouseDown}
+        ref={ref}
+        className={clsx(
+          withBaseName(),
+          { [withBaseName("bottomLabel")]: labels !== "inline" },
+          className
+        )}
+        {...rest}
```

Comment:
Attached rest props here for now - if they want to change the size of the slider all the other components either inherit or fill all the possible spacing using flex / fr.
We could expose the SliderThumb to the user allowing them more flexibility for customisation and accessibility when passing down aria-labels.
Personally i think users want something they can easily use and just pass the props to, so have kept the api consistent with the HTML slide api